src/i18ntestSetup.ts

```diff
@@ -0,0 +1,43 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+i18n.use(initReactI18next).init({
+  lng: 'en',
+  interpolation: {
+    escapeValue: false, // not needed for react as it escapes by default
+  },
+  // common namespace used around the full app
+  ns: ['translationsNS'],
+  resources: {
```

Comment:
Is there a way to use the same resource as we use for the component in prod / dev environment? Otherwise, there could be a mismatch between naming causing tests not to check the same behaviour.

Reply:
Using `i18next-http-backend` I did not find a way. Anyway, as we don't use this now I can import the same files.